app/Resources/assets/js/cftree/view-documentclass.js

```diff
@@ -1411,6 +1409,24 @@ function ApxDocument(initializer) {
                     ;
                 }
             }
+
+            // render license information in case of framework has one.
+            if ("undefined" !== typeof self.licenses && self.licenses.length > 0) {
+                let licenseDoc = self.licenses;
+                let licenseText = licenseDoc[0].licenseText;
+
+                html += '<li class="list-group-item">'
+                + '<strong>License:</strong> '
+                + licenseDoc[0].title;
```

Comment:
We should probably wrap the display of the title with `render.escaped()` for safety.